client/app/lib/providers/computecontroller.coffee

```diff
@@ -997,6 +999,47 @@ module.exports = class ComputeController extends KDController
           @emit 'StacksInconsistent', stack
 
 
+  sharingStackTemplate: (_id, type) ->
+    remote.api.JStackTemplate.one { _id }, (err, template) ->
+      template.setAccess type
```

Comment:
let's remove this helper from here and use where needed as is and rename it as `setTemplateAccessLevel`